### ISSUE TITLE
Support More Fields for CSV Import

### ISSUE THREAD

**Issue by jordan-wright** [gophish/gophish#62]
Right now, we are forcing the user to have fields such as ```First Name``` that are case sensitive and very specific.

It would be ideal if we could be more general, such as supporting just "first", "fname", "firstname", etc.

**Comment by richardontheinternet** [comment]
Yes beautiful addition. After the recon stage of a phishing campaign I formulate user name conventions. If I am sending phishes I would love to add mid email $username to add realism and to aid the attack:

"Hi $FIRST NAME,

I have been looking into recent access for your vpn account and i'd like you to change the password for your vpn account. Your details are;

User name: $USERNAME
Password: **********

Regards,"

The only real way to import 400 usernames is the CSV.

**Comment by jcamil** [comment]
Do we want a static set of supported fields - like 10 fields with a specific name, or do we want it to depend on what the user has entered as the header fields?

**Comment by vishakha-mali** [comment]
Hey,I want to try this issue!! Can u specify where the changes are needed?
